slack-connector/cmd/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"github.com/kyma-incubator/hack-showcase/slack-connector/internal/registration"
+	log "github.com/sirupsen/logrus"
+)
+
+func main() {
+	log.Info("server started")
```

Comment:
Actually there's no server here. I'd log something like that
```suggestion
	log.Info("registration started")
```